src/actions/effect/outline.ts

```diff
@@ -3,6 +3,11 @@ import QualifierValue from "../../qualifier/QualifierValue";
 import Qualifier from "../../qualifier/Qualifier";
 import {prepareColor} from "../../utils/prepareColor";
 
+/**
+ * @description Adds an outline to a transparent image. For examples, see the Image Transformations guide.
```

Comment:
move to end?

Reply:
I'm not sure, I think for functions it makes more sense for it to be first

Maybe it should always be first, even in classes?